Add advise how to activate acceptance tests in ProjectTesting.rst (#2894)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/Testing/ProjectTesting.rst

setups like these can be found in the `ddev documentation
<https://ddev.readthedocs.io/en/stable/users/extend/custom-compose-files/>`_.

> To execute acceptance tests in this installation you've to activate this file, usually it's now appended

Reviewer: ```suggestion
To execute acceptance tests in this installation you have to activate this file, usually it is now appended
```
we try to avoid appreviations as they make texts harder to read

> with the suffix ".inactive" and therefore not used when ddev starts. To activate acceptance test the file

Reviewer: ```suggestion
with the suffix `.inactive` and therefore not used when DDEV starts. To activate acceptance test the file
```
